src/App/pages/LandingPage/Main/FilterFormCinema/FilterFormCinema.js

```diff
@@ -0,0 +1,35 @@
+export function FilterFormCinema({}) {
+    let cinemaData =[{"id":655,"cinemaName":"eiufeiufheuf","adress":{"streetAddress":"6388 Lacus Ct","city":"Kenora"}},
```

Comment:
Since you don't modify the cinemaDate value, use 'const' instead. Also, add spaces after ":", "," and around values inside curly braces.

Reply:
fixed